internal/utils/utils.go

```diff
@@ -0,0 +1,19 @@
+package utils
```

Comment:
nit: Utils is a great package name.  Should this file be called `strings` or something else more specific to the type of util it is?

Reply:
Fair question. My thought was that this was to not overthink that part since it's only one util function for now. And maybe as things are added, the proper "homes" would present themselves. That being said, in this case, I would think the file name for this util function would be `arrays.go` since the input is an array/slice.